src/SelectedBeast.js

```diff
@@ -1,18 +1,19 @@
 import React from "react";
+import Modal from 'react-bootstrap/Modal'
 
-class Modal extends React.Component {
+class SelectedBeast extends React.Component {
   render() {
     return(
       <Modal show={this.props.showModal} onHide={this.props.hideModal}>
```

Comment:
Re: what you said about resizing the Modal based on image size -- 
You could try to add a style line to <Modal> as one way to control the size of the Modal, at least. 

```
<Modal 
style={{ width: 'X' }}
show={this.props.showModal} 
onHide={this.props.hideModal}>
```